frontend/packages/helm-plugin/src/catalog/utils/catalog-utils.tsx

```diff
@@ -107,7 +98,7 @@ export const normalizeHelmCharts = (
         ];
 
         const helmChart = {
-          uid: `${chartRepoName}--${digest}`,
+          uid: `${name}--${version}`,
```

Comment:
This will not be unique when same chart is coming from two different chart repositories. I think we need to use chart URLs to make it always unique. Check `#tmp-odc-helm` channel on slack for related conversation around this.